src/components/Tool/FixCommitmentInfo/FixCommitmentInfo.test.tsx

```diff
@@ -142,21 +149,39 @@ describe('FixCommitmentContact', () => {
         'Are you sure you wish to update Tester 1 commitment info?',
       ),
     ).toBeInTheDocument(),
-      userEvent.click(getAllByTestId('action-button')[1]);
+      userEvent.click(await findByText('Yes'));
```

Comment:
I don't think you need todo a `await find..` as the previous call was a get as well.
```suggestion
      userEvent.click(getByText('Yes'));
```